packages/zowe-explorer/__tests__/__theia__/extension.theia.test.ts

```diff
@@ -257,6 +280,8 @@ describe("Delete Profiles", () => {
         await driverChrome.clickOnZoweExplorer();
     });
 
+    afterEach(screenshotIfFailed(driverChrome));
```

Comment:
Do we need `afterEach` multiple times in this `describe` block?

Reply:
I believe there is only one `afterEach` per `describe` block in this PR, but the way the GitHub diff collapses the lines may make it tricky to review. This `afterEach` applies to the "Delete Profiles" `describe` on line 274.

Reply:
Ah, sorry I should have taken a closer look. GitHub was hiding the `describe` blocks in between each `afterEach` 🤦‍♂️ thanks for clarifying!